ports/espressif/boards/seeed_xiao_esp32_s3_sense/mpconfigboard.mk

```diff
@@ -0,0 +1,17 @@
+USB_VID = 0x303A
+USB_PID = 0x80D1
```

Comment:
My impression is that you still need to ask Seeed for a PID, is that right? This one was just chosen by you.

Reply:
I just emailed someone I know there about USB PIDs for their boards. Will keep this in mind too.

Reply:
You're right @dhalbert. I took the data from that Espressif board and put it in the mpconfigboard.mk header. It's not the definitive solution, but it worked. I contacted the folks at Seeed again. They created a wiki with several tasks and I had taken over this a while ago.

Reply:
I've already solve this, ok?

Reply:
Thanks, could you point to that wiki with assigned PIDs? I will bookmark it.

Reply:
Guys, about VID and PID from Seeed, read this:
https://github.com/adafruit/circuitpython/issues/9507#issuecomment-2298119726

Seeed Xiao ESP32-S3
VID: 0x2886
PID: 0x8056